mmv1/third_party/terraform/framework_models/provider_model.go.erb

```diff
@@ -67,6 +67,11 @@ type ProviderBatching struct {
 	EnableBatching types.Bool   `tfsdk:"enable_batching"`
 }
 
+var ProviderBatchingAttributes = map[string]attr.Type{
```

Comment:
I think we need to import `"github.com/hashicorp/terraform-plugin-framework/attr"`

Reply:
Thanks! My IDE did this in TPGB while I was iterating and forgot to check for changed imports.